frontend/app/src/components/accounts/AccountBalanceTable.vue

```diff
@@ -588,13 +624,23 @@ const fetchDetectedTokensAndQueryBalance = async (address: string) => {
   });
 };
 
+const fetchAllDetectedTokensAndQueryBalance = async () => {
+  const promises = get(visibleBalances).map(async balance => {
+    const address = balance.address;
+    await fetchDetectedTokensAndQueryBalance(address);
+  });
+
+  await Promise.allSettled(promises);
+};
+
```

Comment:
This will fetch all the tokens for all accounts every time anyone clicks on the row button, correct?


@LefterisJP what you wanted was to have an extra option to detect tokens for all the addresses, not for the row to always detect tokens for all the addresses right?

Reply:
There is `header button` and `row button`. `Header-button` fetch all addresses token, `row-button` only fetch token for the address on that row

Reply:
> There is `header button` and `row button`. `Header-button` fetch all addresses token, `row-button` only fetch token for the address on that row

The header button I didn't notice immediately btw because it is the first time we do something like this.

Reply:
hmm okay, I think it's better to move it beside that big `Delete` button on the top?